go.mod

```diff
@@ -28,11 +28,11 @@ require (
 	github.com/urfave/cli v1.22.15
 	go.opencensus.io v0.24.0
 	golang.org/x/net v0.27.0
-	golang.org/x/oauth2 v0.21.0
-	golang.org/x/sync v0.7.0
-	golang.org/x/sys v0.22.0
+	golang.org/x/oauth2 v0.22.0
+    golang.org/x/sync v0.8.0
```

Comment:
Formatting seems off.

Reply:
yes somehow in intellij , this was looking correct, but mod tidy fixed it, thanks for catching !!